x-pack/plugins/observability/public/hooks/use_get_cases_user_permissions.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEffect, useState } from 'react';
+import { useKibana } from '../utils/kibana_react';
+
+export interface UseGetUserCasesPermissions {
+  crud: boolean;
+  read: boolean;
+}
+
+export const useGetUserCasesPermissions = () => {
```

Comment:
There's no linting around it but on APM/Observability we tend to prefer using `export function` instead of `const` for hooks and components. Not a blocker for me but FYI.

Reply:
yeah it linted me on that in a few places but not here, will update